### ISSUE TITLE
Add the floating animaton in the svg of the home page.

### ISSUE THREAD

**Issue by Shivangi-ch** [ADRE9/bunk-manager-mern#140]
**Is your feature request related to a problem? Please describe.**
No its not a problem.

**Describe the solution you'd like**
I would like to add the floating animaton in the svg of the home page. this will make the page look more attractive.

**Additional context**
I am a GSSOC participant .
Please assign this to me.

**Comment by Shivangi-ch** [comment]
@Sanyam2000-dot Please assign this to me.

**Comment by Sanyam2000-dot** [comment]
Hey @Shivangi-ch, may I know what exactly do you want to add?

**Comment by Shivangi-ch** [comment]
@Sanyam2000-dot I would like to add these types of animations in illustrations(svg).
Demo:
![Hnet-imagesdf](https://user-images.githubusercontent.com/57783826/117811159-d9d1e600-b27d-11eb-9ed6-e3b48bb5e0ba.gif)

**Comment by Sanyam2000-dot** [comment]
This looks cool but right now we are redesigning the pages and once that is done you can surely do this.

**Comment by ADRE9** [comment]
Yes exactly it looks cool. Let the page be made you can work on it. @Shivangi-ch .

**Comment by kunaljain0212** [comment]
Any updates on this @Shivangi-ch ?

**Comment by Shivangi-ch** [comment]
@Sanyam2000-dot and @ADRE9 said to wait till the page is redesigned!
@kunaljain0212

**Comment by Shivangi-ch** [comment]
@Sanyam2000-dot @ADRE9 Can I start working on this issue now.
